pkg/controller/node/node_controller.go

```diff
@@ -1173,3 +1179,25 @@ func getErrorString(err error) string {
 	}
 	return ""
 }
+
+// shouldApplyConfigToPool checks if the node-controller should apply configurations to the given pool.
```

Comment:
In openshift/machine-config-operator#3731, there are helper functions for this in `pkg/controller/common/helpers.go`, but lets leave this as-is for now. I'll address it in my PR once this lands.